### PR TITLE
adding public helper and adding custom form name

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

### Added
- custom form name to general settings
- new forms public helper class to be able to provide helpers to a external project 

# Screenshots / Videos
Screenshot 2022-12-09 at 00 38 30

### CODE REVIEW

File: src/Settings/Settings/SettingsGeneral.php [hunk 1]
```diff
@@ -137,6 +142,22 @@ public function getSettingsData(string $formId): array
...
+					[
+						'component' => 'tab',
+						'tabLabel' => \__('Identification', 'eightshift-forms'),
+						'tabContent' => [
+							[
+								'component' => 'input',
+								'inputName' => $this->getSettingsName(self::SETTINGS_GENERAL_FORM_CUSTOM_NAME_KEY),
+								'inputId' => $this->getSettingsName(self::SETTINGS_GENERAL_FORM_CUSTOM_NAME_KEY),
+								'inputFieldLabel' => \__('Form custom name', 'eightshift-forms'),
+								// translators: %s will be replaced with th filter output copy.
```
Comment (iobrado) [inline src/Settings/Settings/SettingsGeneral.php:154]: I think this comment is obsolete. There's no %s to be replaced 😄

**Review by iobrado — APPROVED** [review]

**Review by piqusy — APPROVED** [review]

**Review by dadadavorin — APPROVED** [review]

File: src/Helpers/esForms.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+/**
+ * Outputs the forms custom unique name set in the settings by provided form ID.
+ *
+ * @example echo esFormsGetFormIdByName('22826');
+ *
+ * @param string $formId Form ID.
+ *
+ * @return string
+ */
+function esFormsGetFormIdByName(string $formId): string
```
Comment (mbmjertan) [inline src/Helpers/esForms.php:21]: Where will this be used? Why do we need to define a function in the global namespace?

File: src/Helpers/esForms.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+{
+	$class = (
+		new class() // phpcs:ignore
+		{
+			use SettingsHelper;
+		}
+	);
```
Comment (mbmjertan) [inline src/Helpers/esForms.php:28]: understandable, but 😔

File: src/Helpers/esForms.php (hunk 2 above)
Comment (iruzevic) [reply src/Helpers/esForms.php:21]: in your theme or other plugin

File: src/Helpers/esForms.php (hunk 3 above)
Comment (iruzevic) [reply src/Helpers/esForms.php:28]: I know, not happy also

**Pull request merged**